### ISSUE TITLE
[libigl] build failure with feature <imgui>

### ISSUE THREAD

**Issue by StarGate-One** [microsoft/vcpkg#12442]
**Host Environment**
- OS: [Windows 10 Version 10.0.19041.388]
- Compiler: Visual Studio 2019 Version 16.6.4 Tools 14.26.28801

**To Reproduce**
`./vcpkg install libigl[core,glfw,imgui,png,xml]:x64-windows`

**Failure logs** 
- Cut and past the appropriate build messages from the console output.
`Computing installation plan...`
**`Package imgui does not have a bindings feature`**
- Please attach any additional failure logs mentioned in the console output.
None

**Additional context**
It seems port imgui had feature `bindings` changed to multiple separate bindings (commit https://github.com/microsoft/vcpkg/commit/0b78ddc14bf513ddc33fe70099c5993d789c7081) , making the imgui feature `bindings` invalid.

**Comment by JackBoosY** [comment]
#10252 will fix this issue.

**Comment by StarGate-One** [comment]
Ok, thank you @JackBoosY

**Comment by StarGate-One** [comment]
@JackBoosY @strega-nil 
Please reopen this issue, the problem still exists.

From the most up-to-date git pull as of 20200801 @ 21:50 eastern, where port libigl was pushed, the CONTROL file for port libigl still is requiring imgui[bindings] and the bindings feature for imgui does not exist:

https://github.com/microsoft/vcpkg/blob/b46242ff15f78a2da58548bda04850daf3ddca48/ports/libigl/CONTROL#L20-L22

**Comment by StarGate-One** [comment]
`>vcpkg search imgui`
`imgui                1.77#2           Bloat-free Immediate Mode Graphical User interface for C++ with minimal depend...`
`imgui[allegro5-binding]               Make available Allegro5 binding`
`imgui[dx9-binding]                    Make available DirectX9 binding`
`imgui[dx10-binding]                   Make available DirectX10 binding`
`imgui[dx11-binding]                   Make available DirectX11 binding`
`imgui[dx12-binding]                   Make available DirectX12 binding`
`imgui[glfw-binding]                   Make available GLFW binding`
`imgui[glut-binding]                   Make available Glut binding`
`imgui[marmalade-binding]              Make available Marmalade binding`
`imgui[metal-binding]                  Make available Metal binding`
`imgui[opengl2-binding]                Make available OpenGL (legacy) binding`
`imgui[opengl3-glew-binding]           Make available OpenGL3/ES/ES2 (modern) binding with GLEW`
`imgui[opengl3-glad-binding]           Make available OpenGL3/ES/ES2 (modern) binding with glad`
`imgui[opengl3-gl3w-binding]           Make available OpenGL3/ES/ES2 (modern) binding with gl3w`
`imgui[opengl3-glbinding-binding]      Make available OpenGL3/ES/ES2 (modern) binding glbinding`
`imgui[osx-binding]                    Make available OSX binding`
`imgui[sdl2-binding]                   Make available SDL2 binding`
`imgui[vulkan-binding]                 Make available Vulkan binding`
`imgui[win32-binding]                  Make available Win32 binding`
`imgui[freetype]                       Build font atlases using FreeType instead of stb_truetype`

**`libigl[imgui]                         Build with imgui`** `no bindings feature exists in imgui as the bindings were split into individual features in port imgui`

**Comment by strega-nil** [comment]
Shoot, I missed this. Lemme fix it.

**Comment by StarGate-One** [comment]
@strega-nil 
I think everyone is missing the point I was trying to make.
- port imgui features were changed a few weeks back:
- Prior to this change, to install port was:
   `vcpkg install imgui[core,bindings,freetype]`
- After the change to separate all the combined features in  [bindings] into individual features:
   `vcpkg install imgui[allegro5-binding,core,dx10-binding,dx11-binding,dx12-binding,dx9-binding,freetype,glfw-binding,glut-binding,marmalade-binding,opengl2-binding,opengl3-gl3w-binding,opengl3-glad-binding,opengl3-glbinding-binding,opengl3-glew-binding,sdl2-binding,vulkan-binding,win32-binding]`
-Notice there is no longer a single feature bindings only each individual components.
- port libigl is referencing port imgui[bindings] and imgui[bindings] no longer exists.
- in other words in the **CONTROL** file should look something like this for port libigl requiring bindings feature support:
` Feature: imgui`
`Description: Build with imgui`
`Build-Depends: imgui[allegro5-binding,core,dx10-binding,dx11-binding,dx12-binding,dx9-binding,freetype]` _or something similar as I do not know what libigl should use for the features required from imgui_

**Comment by StarGate-One** [comment]
Looks good to me - thanks for the hard work @strega-nil and @BillyONeal
